docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,62 @@
+# Run Antrea agent simulator
+
+This Documentation describes how to run Antrea agent simulator.
+
+## Build the images
+
+1. Build simulator image
+
+   ```bash
+   make build-scale-simulator
+   ```
+
+2. Build the kubemark image
+
+   If you alreay have a kubemark image, please ignore this step.
+
+   ```bash
+   cd $KUBERNETES_PATH
+   make WHAT=cmd/kubemark KUBE_BUILD_PLATFORMS=linux/amd64
+   cp ./_output/local/go/bin/linux_amd64/kubemark cluster/images/kubemark
+   cd cluster/images/kubemark
+   docker build -t kubemark:latest .
+   ```
```

Comment:
@tnqn I thought you wanted to push the kubemark image to our dockerhub / harbor registry?

Reply:
I have done that, where do you think the commands that generate the image should be put?

Reply:
Not sure we need to include them in the repository at all? If you want to document them somewhere for future updates to the image, I guess you can add a document under https://github.com/vmware-tanzu/antrea/tree/master/docs/maintainers

I mostly wanted to point out that the current steps are misleading:
* users don't need to build the image themselves
* if users do decide to build the image themselves, they also need to load it into the Nodes and change the tag, as the manifest file references `projects.registry.vmware.com/antrea/kubemark:v1.18.4`

Reply:
I removed this part from antrea-agent-simulator.md, and move to docs/maintainer/kubemark.md